File: A's/py/705A.py
def main() -> None:
    n: int = int(input())
    for i in range(n):
        if i % 2 != 0 and i == n - 1: print("I love it", end=""); break
        elif i % 2 == 0 and i == n - 1: print("I hate it ", end=""); break
        elif i % 2 != 0: print("I love that", end=" ")
        else: print("I hate that", end=" ")

if __name__ == "__main__":
    main()

File: A's/cpp/705A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n; cin >> n;
    vector<string> v;
    for (int i = 0; i < n; i++) {
        if (i % 2 != 0 && i == n - 1) { cout << "I love it"; break; }
        else if (i % 2 == 0 && i == n - 1) { cout << "I hate it "; break; }
        else if (i % 2 != 0) cout << "I love that ";
        else cout << "I hate that ";
    }
    
    return 0;
}